doc/HOWTO_RELEASE.rst.txt

```diff
@@ -337,6 +326,22 @@ strides are used in the development branch to smoke out problems. The
 `patch <https://github.com/numpy/numpy/pull/5996>`_ should be updated if
 necessary and applied to the release branch to rationalize the strides.
 
+Build and archive documentation
+-------------------------------
+Do::
+
+    cd doc/
+    make dist
+
+to check that the documentation is in a buildable state. Then, after tagging,
+create an archive of the documentation in the numpy/doc repo::
+
+	make merge-doc
```

Comment:
looks like this block needs indenting to render correctly

Reply:
changed tabs to spaces